geoh5py/objects/surveys/direct_current.py

```diff
@@ -103,13 +103,20 @@ def ab_map(self) -> dict | None:
             return self.ab_cell_id.value_map
         return None
 
-    def copy(self, parent=None, copy_children: bool = True, **kwargs):
+    def copy(
+        self,
+        parent=None,
+        copy_children: bool = True,
+        clear_cache: bool = False,
+        **kwargs,
+    ):
         """
         Function to copy a survey to a different parent entity.
 
         :param parent: Target parent to copy the entity under. Copied to current
             :obj:`~geoh5py.shared.entity.Entity.parent` if None.
         :param copy_children: Create copies of all children entities along with it.
+        :param clear_cache: Clear array attributes after copy.
```

Comment:
**suggestion:** remove ambiguity in the doc: does it clear attributes on `self` or on the `copy`

Also should the doc says clears the attributes _cache_ array?

I do not quite understand where attributes are stored if cleared in memory and the file has not been saved to disk yet,